WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -617,14 +622,14 @@ extension MediaCoordinator: MediaProgressCoordinatorDelegate {
 extension MediaCoordinator: Uploader {
     func resume() {
         let service = MediaService(managedObjectContext: mainContext)
+        addObserverForDeletedFiles()
```

Comment:
Since we're adding the same wildcard observer in `init()`, why do we still need this? 🤔

Reply:
ah, this is a great catch, i meant it to remove it from here. thanks!